connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/entities/ServerInfo.java

```diff
@@ -16,6 +16,7 @@
  */
 package org.apache.kafka.connect.runtime.rest.entities;
 
+import com.fasterxml.jackson.annotation.JsonCreator;
```

Comment:
Please skip this file and review: https://github.com/apache/kafka/pull/6342 instead. Thanks!

Reply:
#6342 was merged. This PR is now rebased on top of it.